package.json

```diff
@@ -333,6 +333,7 @@
     "qrcode.react": "^1.0.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-focus-lock": "^2.9.4",
```

Comment:
Using [`react-focus-lock` library](https://github.com/theKashey/react-focus-lock) to lock focus to the `ModalContent` component or to whatever component uses the `modalContentRef` ref to improve the accessibility for keyboard only and screen reader users